src/Pickles/Pickles.TestFrameworks/XUnit/XUnit2/XUnit2ScenarioOutlineExampleMatcher.cs

```diff
@@ -31,13 +31,15 @@ public class XUnit2ScenarioOutlineExampleMatcher : IScenarioOutlineExampleMatche
         public bool IsMatch(ScenarioOutline scenarioOutline, string[] exampleValues, object scenarioElement)
         {
             var build = this.signatureBuilder.Build(scenarioOutline, exampleValues);
-
             return ScenarioOutlineExampleIsMatch((assembliesAssemblyCollectionTest)scenarioElement, build);
         }
 
         private bool ScenarioOutlineExampleIsMatch(assembliesAssemblyCollectionTest exampleElement, Regex signature)
         {
-            return signature.IsMatch(exampleElement.name.ToLowerInvariant());
+            var testNameWithExample = exampleElement.name;
```

Comment:
My regexes are still rusty: what does the replacement do, exactly?

Reply:
@dirkrombauts : for that one I found an issue with Scenario Outline names and white spaces.
XUnit2ScenarioOutlineExampleMatcher works only in the case when Scenario Outline title does not have white spaces.
Example: 
it works fine for "Scenario Outline: ExampleTestForPayables" or "Scenario Outline: exampletestforpayables"
unfortunately it does not work for: "Scenario Outline: Example Test For Payables"

my regex remove white spaces from the Scenario Outline title and try to match it then.

I did not want to break anything from the existing functionality so I kept existing functionality and try "new match" only if existing one does not pass.
return signature.IsMatch(exampleElement.name.ToLowerInvariant()) || signature.IsMatch(testNameWithExample);
        }